packages/hooks-react/src/useAds.ts

```diff
@@ -33,12 +34,12 @@ export const useAds = ({ mediaId }: { mediaId: string }) => {
 
   const { data: adSchedule, isLoading: isAdScheduleLoading } = useLegacyStandaloneAds({ adScheduleId, enabled: !!adScheduleId });
   const adConfig = useAdConfigFlow
-    ? {
+    ? ({
         client: 'vast',
         schedule: createURL(adScheduleUrls?.xml || '', {
           media_id: mediaId,
         }),
-      }
+      } as AdSchedule)
```

Comment:
Can we rename it to AdConfig and use the following type:

```
export type AdConfig = {
  client: string;
  schedule: string;
};
```

And then we could use AdSchedule instead of DeliveryAdSchedule

Reply:
Sure! I also considered something like this, but didn't changed it because both types are actually ad configs containing schedules.